packages/output-handler/src/index.ts

```diff
@@ -74,11 +80,12 @@ const handleTranscriptionSuccess = async (
 			sesClient,
 			config.app.emailNotificationFromAddress,
 			transcriptionOutput.userEmail,
-			`Transcription complete for ${transcriptionOutput.originalFilename}`,
+			`Transcription ${transcriptionOutput.isTranslation ? 'english translation ' : ''}complete for ${transcriptionOutput.originalFilename}`,
```

Comment:
what do you think about this?
```suggestion
			`${transcriptionOutput.isTranslation ? 'English translation' : 'Transcription'} complete for ${transcriptionOutput.originalFilename}`,
```

Reply:
If it's a translation email we don't want subject to be "Transcription english translation complete..." right?

Reply:
and same for failure subject and message body?

Reply:
hmm I was thinking that it's technically a translation of the transcription, but "transcription english translation is a bit of a mouthfull! I think your suggestion is clearer - it's kinda implicit that it's a transcription

Reply:
updated in https://github.com/guardian/transcription-service/pull/100/commits/47678134842921b085e54defa5dc173899df2c1b